bayesian/api/user_api.py

```diff
@@ -27,22 +27,32 @@ def get_user(user_id):
 
 @user_api.route('', methods=['POST'])
 @login_required
-def create_user():
+def generate_uuid_for_user():
     """Endpoint for creating user details."""
     user_uuid = uuid.uuid4()
-    user_utils.create_user(user_uuid, "SNYK")
     return jsonify(user_id=user_uuid)
 
 
-@user_api.route('/<user_id>', methods=['PUT'])
+@user_api.route('', methods=['PUT'])
 @login_required
-def update_user(user_id):
-    """Endpoint for updating user details."""
+def create_or_update_user():
+    """Endpoint for creating or updating user details."""
+    content = request.json
+    user_id = content['user_id']
```

Comment:
I think `user_id` will throw `KeyError` if user_id is not in request. Can you please check.

Reply:
Done